src/store/modules/user/actions.ts

```diff
@@ -1,11 +1,27 @@
 import { ActionTree } from 'vuex'
 import { Actions, EActionUser, EMutationUser, IStateUser } from '@/store/modules/user/typesUser'
 import { RootStateInterface } from '@/store/types'
+import { getCurrentUserScoreDB, updateUserScore } from '@/api/DBFirebaseHelpers'
+import { state } from '@/store/modules/user/user'
+import { IFirestoreScore } from '@/types'
 
 export const actions: ActionTree<IStateUser, RootStateInterface> & Actions = {
   [EActionUser.SET_USER]: ({
     commit
   }, payload) => {
     commit(EMutationUser.SET_USER, payload)
+  },
+  [EActionUser.GET_DB_SCORE]: async ({
+    commit
+  }) => {
+    const payload: IFirestoreScore[] | undefined = await getCurrentUserScoreDB(state.user?.uid!)
```

Comment:
is it possible that a user doesn't have `uid`?
When it's possible that there is no `user`? I understand that it could be too complex to plan all scenarios at the current stage. It's rather a question to think about